### PR TITLE
RST-1893 - Rails 5.2 update

### PR DESCRIPTION
**Pull request by zaparka**
Ready to merge to master and deploy to PROD

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -14,50 +14,56 @@
...
 
   # Compress JavaScripts and CSS.
   config.assets.js_compressor = :uglifier
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
-
-  # Generate digests for assets URLs.
+  config.assets.compile = true
```
Comment (leoapost) [inline config/environments/production.rb]: All the assets are being precompiled before deployments. Any reasons this needs to be true?

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "repository": {
     "type": "git",
-    "url": "https://github.com/ministryofjustice/fr-staffapp"
+    "url": "git+https://github.com/ministryofjustice/fr-staffapp.git"
```
Comment (leoapost) [inline package.json]: There is a new url for the repo: https://github.com/hmcts/hwf-staffapp

File: lib/tasks/static_pages.rake [hunk 3]
```diff
@@ -2,7 +2,8 @@ namespace :static_pages do
   desc "Generates static pages"
   Rails.application.middleware.tap do |middleware|
     middleware.delete ActiveRecord::Migration::CheckPending
-    middleware.delete ActiveRecord::ConnectionAdapters::ConnectionManagement
+    # not supported anymore - do we need this? #TODO update
+    # middleware.delete ActiveRecord::ConnectionAdapters::ConnectionManagement
```
Comment (leoapost) [inline lib/tasks/static_pages.rake]: Can we delete this if it's not needed?

**Comment by AlexaVB** [comment]
Approved :)

**Pull request merged**